src/main/java/TaskList.java

```diff
@@ -0,0 +1,205 @@
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class TaskList {
+    private ArrayList<Task> database;
```

Comment:
How about using a name that has plural form? Would that be better since this is representing a collection of objects?